api-server-rest/openapi/api.yaml

```diff
@@ -0,0 +1,148 @@
+{
```

Comment:
should the file name be `-.json`?

Reply:
Thanks @Xynnn007 , good catch, fixed now：）